docs/visualize/lens.asciidoc

```diff
@@ -66,46 +57,40 @@ image::images/lens_data_info.png[]
 [[change-the-visualization-type]]
 ==== Change the visualization type
 
-With Lens, you are no longer required to build each visualization from scratch. Lens allows
-you to switch between any supported chart type at any time. Lens also provides
-suggestions, which are shortcuts to alternate visualizations based on the data you have.
+*Lens* enables you to switch between any supported visualization type at any time with *Suggestions*.
```

Comment:
I want to distinguish between the "chart switcher" and the "suggestions" even though they are implemented in a similar way.

```suggestion
*Lens* enables you to switch between any supported visualization type at any time.
```